pynicotine/gtkgui/ui/search.ui

```diff
@@ -159,14 +159,14 @@
                 <property name="margin-start">12</property>
                 <property name="margin-end">6</property>
                 <child>
-                  <object class="GtkFlowBox">
+                  <object class="GtkFlowBox" id="FiltersFlowBox">
```

Comment:
No need for these new ids.

Reply:
Okay I shall remove them. My future intention is to provide some sort of mechanism for showing/hiding the visibility to allow the filters bar to be customized.